app/screens/full-onboarding-flow/full-onboarding-flow.tsx

```diff
@@ -0,0 +1,168 @@
+} from "@app/graphql/generated"
+import { AccountLevel, useLevel } from "@app/graphql/level-context"
+import { useI18nContext } from "@app/i18n/i18n-react"
+import { isIos } from "@app/utils/helper"
+import Onfido, { OnfidoTheme } from "@onfido/react-native-sdk"
+import { Input, Text, makeStyles } from "@rneui/themed"
+import React, { useState } from "react"
+import { Alert, View } from "react-native"
+
+gql`
+  mutation onboardingFlowStart($input: OnboardingFlowStartInput!) {
+    onboardingFlowStart(input: $input) {
+      workflowRunId
+      tokenAndroid
+      tokenIos
+    }
+  }
+
+  query debugScreen {
+    me {
+      id
+      defaultAccount {
+        id
+      }
+    }
+  }
+`
+
+export const FullOnboardingFlowScreen: React.FC = () => {
+  const { LL } = useI18nContext()
+
+  const styles = useStyles()
+  styles
+
+  const { currentLevel } = useLevel()
+
+  const { data: dataDebug } = useDebugScreenQuery()
+  const accountId = dataDebug?.me?.defaultAccount?.id
+
+  const [onboardingFlowStart] = useOnboardingFlowStartMutation()
+
+  const [firstName, setFirstName] = useState("")
+  const [lastName, setLastName] = useState("")
+
+  const confirmNames = async () => {
+    Alert.alert(
+      LL.FullOnboarding.confirmNameTitle(),
+      LL.FullOnboarding.confirmNameContent({ firstName, lastName }),
+      [
+        { text: LL.common.cancel(), onPress: () => {} },
+        {
+          text: LL.common.yes(),
+          onPress: onfidoStart,
+        },
+      ],
+    )
+  }
+
+  const onfidoStart = async () => {
+    if (!accountId) {
+      console.log("no account id")
+      return
+    }
+
+    const res = await onboardingFlowStart({
+      variables: { input: { accountId, firstName, lastName } },
+    })
+
+    const workflowRunId = res.data?.onboardingFlowStart?.workflowRunId
+    if (!workflowRunId) {
+      Alert.alert("no workflowRunId")
+      return
+    }
+
+    const tokenAndroid = res.data?.onboardingFlowStart?.tokenAndroid
+    const tokenIos = res.data?.onboardingFlowStart?.tokenIos
+
+    const sdkToken = isIos ? tokenIos : tokenAndroid
+
+    if (!sdkToken) {
+      Alert.alert("no sdkToken")
+      return
+    }
+
+    try {
+      /* eslint @typescript-eslint/ban-ts-comment: "off" */
+      // @ts-expect-error
+      const res = await Onfido.start({
+        sdkToken,
+        theme: OnfidoTheme.AUTOMATIC,
+        workflowRunId,
+      })
+
+      Alert.alert(LL.FullOnboarding.success())
+      console.log(res, "success")
+    } catch (err) {
+      Alert.alert(LL.FullOnboarding.error())
+      console.log(err, "error")
+    }
+  }
+
+  if (currentLevel === AccountLevel.Two) {
+    return (
+      <Screen
+        preset="scroll"
+        keyboardShouldPersistTaps="handled"
+        keyboardOffset="navigationHeader"
+      >
+        <Text style={styles.textHeader}>
+          {LL.FullOnboarding.accountVerifiedAlready()}
+        </Text>
+      </Screen>
+    )
+  }
+
+  return (
+    <Screen
```

Comment:
It would be nice to lay out this screen similar to how we style other screens. Here is how its styled in the send flow. It uses padding of 20, and then the PrimaryButton is aligned with the bottom of the screen. https://github.com/GaloyMoney/galoy-mobile/blob/b60b1e086f3b3a4f47caf41f9a00dd010006bc28/app/screens/send-bitcoin-screen/send-bitcoin-details-screen.tsx#L643